frontend/src/features/admin-form/settings/components/EmailNotificationsSection/StatusTrackerToggle.tsx

```diff
@@ -0,0 +1,52 @@
+import { useCallback, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Skeleton } from '@chakra-ui/react'
+
+import { FormSettings } from '~shared/types'
+
+import Toggle from '~components/Toggle'
+
+import { useMutateFormSettings } from '../../mutations'
+import { useAdminFormSettings } from '../../queries'
+
+export const StatusTrackerToggle = (): JSX.Element => {
+  const { t } = useTranslation()
+  const { data: settings, isLoading: isLoadingSettings } =
+    useAdminFormSettings()
```

Comment:
Suggestion: We can take opportunity to make `useAdminFormSettings` accept generics in `frontend/src/features/admin-form/settings/queries.ts`. This can help to remove the need to utilise the helper function `getHasStatusTracker()`. 

E.g.,
```diff
- export const useAdminFormSettings = (): UseQueryResult => {
+ export const useAdminFormSettings = <
+   T extends FormSettings,
+ >(): UseQueryResult<T> => {
```

This then allows us to (1) narrow the type of `FormSettings` returned
```suggestion
    useAdminFormSettings<MultirespondentFormSettings>()
```

and then subsequently be able to access `hasStatusTracker` as if `settings` is now `typeof MultirespondentFormSettings`
```
const hasStatusTracker = Boolean(settings?.hasStatusTracker)
```

Reply:
Nice. I've updated this & update where its implemented in `StatusTrackerToggle`